MT-CTF/capturetheflag
# [no squash] HUD score improvements

This PR is half-maintenance, and half-bugfix:

- `hud_score` has been moved from the `pvp` modpack to `other` modpack. This isn't relevant to PvP in anyway, and can be used for displaying any value, really (see also: #642).
- Existing HUD score elements are reused when adding new ones with matching names. This fixes a bug where a HUD score element can be tricked into believing it's a frickin' neon sign.

Tested; works. Could do with some testing with multiple HUD score elements.

### To test

- Run two instances of Minetest in a server-client configuration.
- Launch CTF, make both players join the same team (`/team join <playername> <teamname>` is your friend).
- One player needs to be severely low on health, and the other needs to be a medic.
- Heal the damaged player using the medic, and observe that the heal score is static, and doesn't flicker all over the place (this is what happens in `master`).
